tests/cue_test.c

```diff
@@ -66,7 +66,7 @@ int test_cues (const char *filename)
 }
 
 
-int main (int argc, char **argv)
+int main ()
```

Comment:
That is not a valid C function definition. It should be:
```
int main (void)
```
or to match the conventions in the rest of the project:
```
int
main (void)
```